frontend/src/js/common-ui/DrawerTitle.tsx

```diff
@@ -27,7 +27,7 @@ interface DrawerTitleProps {
 export const DrawerTitle: FunctionComponent<DrawerTitleProps> = ({ onClose, onLinkCopy, postTitle, preCloser, title }) => (
   <div className="flexbox margin-bottom-small space-between">
     <div className="flexbox center-aligned">
-      <h3 className="capitalized-start flexbox center-aligned">{title}</h3>
+      <h3 className="capitalized-start">{title}</h3>
```

Comment:
Unfortunately this won't work for device, deployment & release details 😬.
An "interesting" restriction to only allow this for `block` level elements... I can't think of a nice way to support both usages (`title` as string or component) so I'm leaning more towards adopting your previous approach in the use cases that don't already provide a "capitalized start title". What do you think?

Reply:
Works for me, on the other hand, having non-inline elements inside of h4 is not the best approach as well. IMO it's a minor issue and we can improve it along the way